<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>AccountReg</class>
 <widget class="QDialog" name="AccountReg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Register Account</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_account" >
     <property name="title" >
      <string>Account</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="le_jid" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lb_jid" >
        <property name="text" >
         <string>Jabber ID:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="TextLabel2" >
        <property name="text" >
         <string>Confirm Password:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="le_confirm" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="le_pass" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TextLabel1" >
        <property name="text" >
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="lb_example" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Example: juliet@capulet.com</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_proxy" >
     <property name="title" >
      <string>Proxy</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_proxychooser" >
        <property name="text" >
         <string>proxychooser</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_advanced" >
     <property name="title" >
      <string>Advanced</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ck_host" >
        <property name="text" >
         <string>Manually Specify Server Host/Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lb_host" >
          <property name="text" >
           <string>Host:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_host" />
        </item>
        <item>
         <widget class="QLabel" name="lb_port" >
          <property name="text" >
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_port" >
          <property name="minimumSize" >
           <size>
            <width>56</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>56</width>
            <height>32767</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_ssl" >
          <property name="text" >
           <string>SSL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_legacy_ssl_probe" >
        <property name="text" >
         <string>Legacy SSL Probe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="BusyWidget" name="busy" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="IconButton" name="pb_close">
		  <property name="text" >
			<string>Close</string>
		  </property>
	  </widget>
     </item>
     <item>
      <widget class="IconButton" name="pb_reg">
		  <property name="text" >
			<string>Register</string>
		  </property>
	  </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>BusyWidget</class>
   <extends>QWidget</extends>
   <header>busywidget.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>iconbutton.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_jid</tabstop>
  <tabstop>le_pass</tabstop>
  <tabstop>le_confirm</tabstop>
  <tabstop>ck_ssl</tabstop>
  <tabstop>ck_host</tabstop>
  <tabstop>le_host</tabstop>
  <tabstop>le_port</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
